src/wp-includes/class-wp-post-type.php

```diff
@@ -962,8 +962,8 @@ public static function get_default_labels() {
 		self::$default_labels = array(
 			'name'                     => array( _x( 'Posts', 'post type general name' ), _x( 'Pages', 'post type general name' ) ),
 			'singular_name'            => array( _x( 'Post', 'post type singular name' ), _x( 'Page', 'post type singular name' ) ),
-			'add_new'                  => array( __( 'Add New Post' ), __( 'Add New Page' ) ),
-			'add_new_item'             => array( __( 'Add New Post' ), __( 'Add New Page' ) ),
+			'add_new'                  => array( __( 'Add Post' ), __( 'Add Page' ) ),
+			'add_new_item'             => array( __( 'Add Post' ), __( 'Add Page' ) ),
```

Comment:
Makes me wonder why we have 'add_new' and 'add_new_item' in the first place?

Reply:
They used to be different strings. After https://core.trac.wordpress.org/changeset/56515 (see also related Trac ticket) they now use the same string and kept for backwards compatibility.